Origin: commit:1c19bcd3fcac15ed337ec8f93663de91045c447e
Author: David Lamparter <equinox@diac24.net>
Subject: libyang BUGFIX don't access LYS_EXT ->parent
Last-Update: 2018-12-05

In lys_is_disabled() we must stop iterating to ->parent when we are on a
LYS_EXT node because its parent pointer is not a lys_node.  This fixes
the Debian unit test failures on big-endian systems.

diff --git a/src/tree_schema.c b/src/tree_schema.c
index b2e39a7dc195..ad0dd4a05c75 100755
--- a/src/tree_schema.c
+++ b/src/tree_schema.c
@@ -108,6 +108,8 @@ check:
             /* unresolved augment, let's say it's enabled */
             return NULL;
         }
+    } else if (node->nodetype == LYS_EXT) {
+        return NULL;
     } else if (node->parent) {
         node = node->parent;
     } else {
-- 
2.19.2

